code/modules/food_and_drinks/food/customizables.dm

```diff
@@ -336,6 +336,12 @@ do {\
 
 
 /obj/item/food/snacks/customizable/attackby(obj/item/I, mob/user, params)
+	if(is_pen(I))
+		var/t = rename_interactive(user, I, use_prefix = FALSE)
```

Comment:
better var name pretty please?

Reply:
Yeah, didn't even glance at that when I copied this code over from weapon renaming.  Fixed!